packages/paste-core/primitives/modal-dialog/src/index.tsx

```diff
@@ -1,22 +1,20 @@
-/**
- * If you’re wondering why we wrapped that package into our own, we reasoned that it
- * would be best for our consumers’ DX, with reasons such as:
- *
- * - If we want to migrate the underlying nuts and bolts in the future, Twilio products
- *   that depend on this primitive would need to replace all occurrences of
- *   `import … from ‘@reach/dialog’` to `import … from ‘@some-new/package’`.
- *   By wrapping it in `@twilio-paste/modal-dialog-primitive`, this refactor can be avoided.
- *   The only change would be a version bump in the package.json file.
- * - We can more strictly enforce semver and backwards compatibility than some of
- *   our dependencies.
- * - We can control when to provide an update and which versions we allow, to
- *   help reduce potential bugs our consumers may face.
- * - We can control which APIs we expose. For example, we may chose to enable or
- *   disable usage of certain undocumented APIs.
- */
+/* Keeping the Reach export until we transition Modal to use Reakit */
 import {
   DialogContent as ModalDialogPrimitiveContent,
   DialogOverlay as ModalDialogPrimitiveOverlay,
 } from '@reach/dialog';
 
 export {ModalDialogPrimitiveContent, ModalDialogPrimitiveOverlay};
+
+export type {
```

Comment:
Waitaminute... we aren't replacing reach dialog in this PR, we're just adding Reakit dialog. I don't think this is a major change?  I do wonder if we should do the full swap in one go though, as having two dialog libraries is kind of odd maybe?

Reply:
For this PR, I didn't want to rebuild Modal yet, so I'm keeping the Reach export there until we rebuild Modal in a separate PR. As part of that one, I'd remove Reach export and dep.